.github/workflows/set_wheel_build_matrix.yml

```
@@ -17,6 +17,12 @@ on:
       cpu_exec_model:
         description: "Kokkos' cpu execution model."
         value: ${{ jobs.set-builder-matrix.outputs.cpu_exec_model }}
+      osx_exec_model:
+        description: "Kokkos' osx execution model."
+        value: ${{ jobs.set-builder-matrix.outputs.osx_exec_model }}
```

OSX (or OS 10) is now called MacOS as of version 11 and above (currently at 13). I think we can change this here and throughout.

```suggestion
      macos_exec_model:
        description: "Kokkos' MacOS execution model."
        value: ${{ jobs.set-builder-matrix.outputs.macos_exec_model }}
```